package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "7.4.8",
+  "version": "7.4.9",
```

Comment:
dont want to block this now, but this should be 7.5.0 now

Reply:
Not a big deal. I'll update it.

Reply:
Thanks